mdoc/src/main/scala/mdoc/internal/cli/Feedback.scala

```diff
@@ -9,4 +10,24 @@ object Feedback {
       s"  --in=$in\n" +
       s"  --out=$out"
   }
+  def mustBeNonEmpty(what: String): String = {
+    s"--$what must be non-empty. To fix this problem, add an argument for the `--$what <value>` argument."
```

Comment:
```suggestion
    s"--$what must be non-empty. To fix this problem, add a value for the `--$what <value>` argument."
```
I sort of think saying `add an argument for argument` is a bit too much argument. What do you think?

Reply:
Good point, fixed